action.yml

```diff
@@ -28,18 +37,27 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - id: go-critic
+      run: echo "version=v0.8.1" >> $GITHUB_OUTPUT
+      shell: bash
     - id: go-junit-report
       run: echo "version=v2.0.0" >> $GITHUB_OUTPUT
       shell: bash
     - id: gomock
       run: echo "version=v1.6.0" >> $GITHUB_OUTPUT
       shell: bash
+    - id: gosec
+      run: echo "version=v2.16.0" >> $GITHUB_OUTPUT
+      shell: bash
     - id: packer
       run: echo "version=1.8.2" >> $GITHUB_OUTPUT
       shell: bash
     - id: shfmt
       run: echo "version=v3.5.1" >> $GITHUB_OUTPUT
       shell: bash
+    - id: staticcheck
+      run: echo "version=2023.1.3" >> $GITHUB_OUTPUT
```

Comment:
Why was this version changed? The old version appears to have worked fine in https://github.com/cisagov/skeleton-generic/pull/138 per the [run log](https://github.com/cisagov/skeleton-generic/actions/runs/5536385486/jobs/10103987744#step:14:41).

Reply:
There was some confusion in regards to which version to reference but I went ahead and used the one shown in the documentation as opposed to the one linked. Previously, 'v0.4.3' was failing so I changed the version number. Maybe it was unrelated.

Reply:
I believe it was unrelated and I would prefer the semver tag. That provides clearer information about what kinds of changes to expect when updating versions.

Reply:
I also prefer the semver tag.

Reply:
Please see https://github.com/cisagov/setup-env-github-action/commit/c625d814f0afdbed58563decc5ad19bc2fdabba0